src/components/capture/create/DetailsForm.tsx

```diff
@@ -1,70 +1,69 @@
 import { materialCells } from '@jsonforms/material-renderers';
 import { JsonForms } from '@jsonforms/react';
-import { Alert, Skeleton, Stack } from '@mui/material';
+import { Alert, Stack } from '@mui/material';
 import useCaptureCreationStore, {
     CaptureCreationState,
 } from 'components/capture/create/Store';
-import { ConnectorTypes } from 'endpoints/connectors';
-import useConnectors from 'hooks/useConnectors';
-import { useEffect, useState } from 'react';
+import { useMemo } from 'react';
 import { FormattedMessage, useIntl } from 'react-intl';
 import {
     defaultOptions,
     defaultRenderers,
     showValidation,
 } from 'services/jsonforms';
 
-type NewCaptureDetailsProps = {
+interface NewCaptureDetailsProps {
     displayValidation: boolean;
     readonly: boolean;
-};
+    connectorTags: any[];
+}
 
 const stateSelectors = {
     formData: (state: CaptureCreationState) => state.details.data,
     setDetails: (state: CaptureCreationState) => state.setDetails,
-    setHasConnectors: (state: CaptureCreationState) => state.setHasConnectors,
+    setConnectors: (state: CaptureCreationState) => state.setConnectors,
 };
 
 function NewCaptureDetails(props: NewCaptureDetailsProps) {
```

Comment:
**Pattern Proposal:** Whenever possible, props should be destructured in the argument list of the function header.

Reply:
This sounds good to me.